drivers/timer/rcar_cmt_timer.c

```diff
@@ -0,0 +1,137 @@
+/*
```

Comment:
Just double-checking: have you verified that this passes all the relevant kernel tests in real hardware? (e.g. `tests/kernel/timer`).

Reply:
These three timer test do pass: 
timer_api
timer_error_case
timer_monotonic

the kernel/timer/starve do not.
have to investigate

Reply:
Do `kernel/timer/starve` and the rest of the tests pass now?

Reply:
I finally succeed to run twister with success on another branch, this one lacks openocd support for west and serial output.
I'm running starve test right now, since this one is out of twister default test suite.

Reply:
kernel/timer/starve does pass